iMichaelOwolabi/my-diarium-backend
# update project README

### What does this PR do?
* update project README
* Remove graphql config from the project

#### How should this be manually tested?
* Clone the repository
* checkout to this branch "ch-update-readme"
* Run the following from the terminal:
- ```npm install```
- ```npm run dev```

You'll see the updated README

### Automated test
* Run ```npm test```

### Any background context you want to provide?
Nil
